dialogue-test-common/src/main/java/com/palantir/dialogue/AbstractSampleServiceClientTest.java

```diff
@@ -107,7 +106,7 @@ public abstract class AbstractSampleServiceClientTest {
 
     @BeforeEach
     public void before() {
-        server.useHttps(SslSocketFactories.createSslSocketFactory(SSL_CONFIG), false);
```

Comment:
the removed argument was disabling tunnel proxy: 

https://github.com/square/okhttp/blob/parent-4.10.0/mockwebserver/src/main/kotlin/okhttp3/mockwebserver/MockWebServer.kt#L276-L280
> `tunnelProxy true to expect the HTTP CONNECT method before negotiating TLS.`